lib/metadata/TableSchema.js

```diff
@@ -49,8 +49,8 @@ class TableSchema {
       const name = RdfUtils.findValue(this.dataset, node, this.ns.name) || titles[0]
       const aboutUrl = RdfUtils.findValue(this.dataset, node, this.ns.aboutUrl)
       const language = RdfUtils.findValue(this.dataset, node, this.ns.lang)
-      const nullValue = RdfUtils.findValue(this.dataset, node, this.ns.null)
-      const defaultValue = RdfUtils.findValue(this.dataset, node, this.ns.default)
+      const nullValue = RdfUtils.findValue(this.dataset, node, this.ns.null) || ''
+      const defaultValue = RdfUtils.findValue(this.dataset, node, this.ns.default) || ''
```

Comment:
The default for `csvw:null` and `csvw:default` should be an empty string as per [the spec](https://www.w3.org/TR/2015/REC-tabular-metadata-20151217/#cell-default)